src/Middleware/WorkerMiddleware.php

```diff
@@ -43,24 +43,24 @@ class WorkerMiddleware
     private $container;
 
     /**
-     * Map task names to a middleware. For instance:
+     * Map event names to a middleware. For instance:
      *
      * [
-     *      'process_image' => ProcessImageMiddleware::class
+     *      'image.saved' => ProcessImageMiddleware::class
```

Comment:
S single middleware is ok for my use-case, but that could be a nice feature :+1: